src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -427,16 +428,14 @@ import { EnumUtilities, LegalServices } from '@/services/'
     AllowableActionsMixin,
     DateMixin,
     FilingMixin
-  ]
+  ],
+  computed: {
+    ...mapGetters(['getEditUrl', 'getFilings', 'hasCourtOrders', 'isBenBcCccUlc', 'isFirm', 'isRoleStaff'])
+  }
```

Comment:
Do getters declared this still way still have a type? That was an advantage of the `@Getter` decorator. (You may be able to declare type in the mapGetters array items.)

Reply:
I switched `@getters` to `...mapGetters()` because you previously commented in [pull 450](https://github.com/bcgov/business-filings-ui/pull/450)
> I'm OK with this format (instead of using @Getter decorators. I think this is more in line 
> with [Vue3 Composition API](https://vuejs.org/guide/extras/composition-api-faq.html) anyway.

Reply:
I did write that. Hmm.

What do you think -- are we losing some helpful typings this way? Is there a solution?

Reply:
Since typing is important, I think we should return the syntax we were using previously:
`@Getter isRoleStaff!: boolean`

It's slightly more verbose and we run the risk of a mismatch between the return type from the store and the type as defined in a component.

Reply:
If you agree, let me know and I'll reverse some of the changes I made in this PR.

Reply:
I agree. Sorry about that.